package/libwebsockets/libwebsockets.mk

```
@@ -76,4 +76,9 @@ ifeq ($(BR2_SHARED_LIBS),y)
 LIBWEBSOCKETS_CONF_OPTS += -DLWS_WITH_STATIC=OFF
 endif
 
+ifeq ($(BR2_PACKAGE_SYSTEMD),y)
+LIBWEBSOCKETS_DEPENDENCIES += systemd
+LIBWEBSOCKETS_CONF_OPTS += -DLWS_WITH_SDEVENT=1
+endif
+
```

Would move this up above the SHARED_LIBS to line 74 since this is the something as LIBUV and copy the same setup

```suggestion
ifeq ($(BR2_PACKAGE_SYSTEMD),y)
LIBWEBSOCKETS_DEPENDENCIES += systemd
LIBWEBSOCKETS_CONF_OPTS += -DLWS_WITH_SDEVENT=ON
else
LIBWEBSOCKETS_CONF_OPTS += -DLWS_WITH_SDEVENT=OFF
endif

```

---

Fine moving it up, but the else condition isn't needed because it defaults to `OFF`.  Adding it for explicit safety?